GoogleCloudPlatform/cloud-foundation-toolkit
# github workflow for bumping TF/gcloud versions and build/push new image

**Github Actions workflows**

**-  that creates a new PR with latest versions for Terraform and gcloud sdk in the makefile.** 
Steps include: 

- check if TF is latest; if not update in Makefile
- check if gcloud is latest; if not update in Makefile
- if any changes exist for Makefile create commit
- open PR on branch `create-pull-request/patch-tools-version`
- scheduled via cron` 0 2 * * *`

Example PR:
![image](https://user-images.githubusercontent.com/10992720/78467589-ca27bb80-76d3-11ea-8590-93d2b3d3a2b6.png)


Example Run:
https://github.com/bharathkkb/cloud-foundation-toolkit/actions/runs/70940195

**- push new cft developer image to gcr if any files change in`infra/build/`**

Steps include: 

- Build latest image
- Push latest image

Example Run:
https://github.com/bharathkkb/cloud-foundation-toolkit/actions/runs/70940194
